### ISSUE TITLE
nodejs LTS 20.9

### ISSUE THREAD

**Issue by njames** [cap-js/docs#509]
I noticed that LTS is [now 20.9](https://nodejs.org/en/blog/release/v20.9.0) 

What's the plan to update the requirement in CAP and other tools so I dont have a swag of `EBADENGINE` warnings ?

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@sap/approuter@14.3.3',
npm WARN EBADENGINE   required: { node: '^16.0.0 || ^18.0.0' },
npm WARN EBADENGINE   current: { node: 'v20.9.0', npm: '10.2.1' }
npm WARN EBADENGINE }
```

**Comment by renejeglinsky** [comment]
Hi Nigel,
I can't really tell when individual components will be available with Node 20.9. I can assure you, that we are aware and that we are pushing for adaptation. I'm closing this issue as we can't track all @sap-scoped modules here.

Thanks,
René

**Comment by njames** [comment]
Thanks  - that sounds marvellous.
